week-1/implement/dedupe.js

```diff
@@ -1 +1,7 @@
-function dedupe() {}
+let arr = [1, 2, 2, 3,"a","a","b","c"];
+function dedupe(arr) {
+    return arr.filter((item,
+        index) => arr.indexOf(item) === index);
+}
+
+console.log(dedupe(arr));
```

Comment:
You have not exported the function of the file. Also, you have not written a ny test cases to test your functionality!